### PR TITLE
bump versions to 3.0

### PR DESCRIPTION
**Pull request by khieta**
*Issue #, if available:*

*Description of changes:*

This PR fixes the CI break caused by [cedar#334](https://github.com/cedar-policy/cedar/pull/334), as well as several other minor things I noticed when checking that I was able to follow build instructions locally. Whitespace changes are due to auto-formatting.

**Note:** I expect to followup soon with a PR to create a `release/2.x` branch, per #60. That PR will undo some of these changes.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cedar-rust-hello-world/src/main.rs [hunk 1]
```diff
@@ -376,8 +376,7 @@ fn validate() {
                         "type": "Record",
                         "attributes": {
                             "age": {
-                                "type": "Long",
-                                "name": "age"
+                                "type": "Long"
```
Comment (khieta) [inline cedar-rust-hello-world/src/main.rs:379]: Was causing a panic with `cargo run`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -99,6 +100,9 @@ jobs:
       - name: Build cedar-java-hello-world
         working-directory: ./cedar-java-hello-world
         run: bash config.sh && ./gradlew build
+      - name: Run cedar-java-hello-world
+        working-directory: ./cedar-java-hello-world
+        run: ./gradlew run
```
Comment (andrewmwells-amazon) [inline .github/workflows/ci.yml:105]:
```suggestion
        run: bash config.sh && ./gradlew run
```

Comment (khieta) [reply .github/workflows/ci.yml]: I think the issue is actually this line in the `build.gradle` file:
```
application {
  mainClassName = 'samplelib.SampleRunner'
}
```

`SampleRunner` isn't actually a class (`SampleJavaClass` is, but it has no main method). I think the intention was to run the example code via the tests, so I've changed this to `./gradlew test`. Possibly I will also need the config step -- will try that if the build fails.

**Review by shaobo-he-aws — APPROVED** [review]

**Review by andrewmwells-amazon — APPROVED** [review]

**Comment by khieta** [comment]
Merging this PR for now to get the CI fixed for our other repos. I expect that we'll also want the changes in #63, but will review that separately.

**Pull request merged**